app/src/js/components/viewer2d.tsx

```diff
@@ -208,19 +208,39 @@ export class Viewer2D extends DrawableViewer<Viewer2DProps> {
     const newConfig = { ...config }
     if (newScale >= MIN_SCALE && newScale <= MAX_SCALE) {
       newConfig.viewScale = newScale
-    } else {
-      zoomRatio = 1
+
```

Comment:
Could you add a comment here to explain what this change is doing? A little confused about the case in which displayTop and displayLeft need to be set to 0